dist/windows/installer-translations/basque.nsi

```diff
@@ -6,6 +6,8 @@ LangString inst_qbt_req ${LANG_BASQUE} "qBittorrent (beharrezkoa)"
 LangString inst_dekstop ${LANG_BASQUE} "Sortu Mahaigaineko Lasterbidea"
 ;LangString inst_startmenu ${LANG_ENGLISH} "Create Start Menu Shortcut"
 LangString inst_startmenu ${LANG_BASQUE} "Sortu Hasierako Lasterbidea"
+;LangString inst_startup ${LANG_ENGLISH} "Start qBittorrent on Windows start up"
+LangString inst_startup ${LANG_BASQUE} "Hasi qBittorrent abiarazten leihoetan"
```

Comment:
IIRC, you use existing translations, isn't it?

Reply:
Used google translator. Any issues?

Reply:
>Any issues?

I don't know. I don't speak all these languages.
>Used google translator.

I think it can be used for the first time. But aren't appropriate strings exist in qBittorrent translation files? The same text is used in qBittorrent Options dialog so translations should be there.

Reply:
I looked into the translation files and most of them are unfinished. They don't contain a translation for this string. So it would be a waste of time to go through 58 different translations whereas 90% of those don't contain a valid translation for this. So I chose to use Google translator. Also FYI, only 23 out of these 58 translations are actual translations. Others are just using the English strings.